panel/template/golden/golden.html

```diff
@@ -0,0 +1,69 @@
+{% extends base %}	
+
+<!-- goes in body -->	
+{% block postamble %}	
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.11.1.min.js"></script>	
+<script type="text/javascript" src="https://golden-layout.com/files/latest/js/goldenlayout.js"></script>	
+<link type="text/css" rel="stylesheet" href="https://golden-layout.com/files/latest/css/goldenlayout-base.css" />	
+{% endblock %}	
+
+<!-- goes in body -->	
+{% block contents %}	
+<header class="app-bar" id="header">	
+  <div class="">	
+    <section class="">	
+      <span class="header-title">{{ app_title }}</span>	
+	  {% for doc in docs %}	
+      {% for root in doc.roots %}	
+      {% if "header" in root.tags %}	
+      {type: 'component', componentName: 'view', componentState: {model: '{{ embed(root) }}', title: "{{ 'Sidebar' }}"}},	
+      {% endif %}	
+      {% endfor %}	
+      {% endfor %}	
+    </section>	
+  </div>	
+</header>	
+
+<div class="main-area header-adjust" id="main">	
+  <main class="main-content" id="main-content"></main>	
+</div>	
+
+<script>	
+  var config = {	
+    content: [
+        {
+            type: 'row', content: [
+                {type: 'component', componentName: 'view', componentState: {width: 300, title: "Controls", model: '{% for doc in docs %}{% for root in doc.roots %}{% if "nav" in root.tags %} {{ embed(root) }} {% endif %}{% endfor %}{% endfor %}'}},       
+                {
+                    type: 'stack', content: [	
+                        {% for doc in docs %}	
+                        {% for root in doc.roots %}	
+                        {% if "main" in root.tags%}	
+                        {type: 'component', componentName: 'view', componentState: {model: '{{ embed(root) }}', title: "{{ 'Content' }}" }},	
+                        {% endif %}	
+                        {% endfor %}	
+                        {% endfor %}	
+                    ]
+                }
+            ]
+        }
+    ]	
+  };	
+
+  var myLayout = new GoldenLayout(config, $('#main-content'));	
+  myLayout.registerComponent('view', function( container, componentState ){	
+    const {width, css_classes} = componentState
+    if(width)	
+      container.on('open', () => container.setSize(width, container.height))	
+    if (css_classes)	
+      css_classes.map((item) => container.getElement().addClass(item))
+    container.setTitle(componentState.title)
+    container.getElement().html(componentState.model);
+    container.on('resize', () => window.dispatchEvent(new Event('resize')))	
+    window.dispatchEvent(new Event('resize'));	
```

Comment:
I noticed a problem of event resize recursion when a container is maximized
To avoid it I use this code
```javascript
var containers = []
var resize_dispatcher = () => window.dispatchEvent(new Event("resize"))
myLayout.registerComponent("view", function (container, componentState) {
  const {width, css_classes} = componentState
  if (width)
    container.on("open", () => container.setSize(width, container.height))
  if (css_classes)
    css_classes.map((item) => container.getElement().addClass(item))
  container.setTitle(componentState.title)
  container.getElement().html(componentState.model)
  container.on("resize", resize_dispatcher)
  containers.push(container)
  container.on("maximised", () => {
    containers.forEach((container) =>
      container.off("resize", resize_dispatcher)
    )
  })
  container.on("minimised", () => {
    resize_dispatcher()
    containers.forEach((container) =>
      container.on("resize", resize_dispatcher)
    )
  })
})
myLayout.init()
```